### PR TITLE
fix string in tuple for python 3

### PR DESCRIPTION
**Pull request by pmuston**
I got an exception when decoding a string in a tuple in Python 3.
try:
int_str_int = Tuple(Integer, String, Integer)
encoded = int_str_int.encode((123, 'abc\ndef', 456,))
decoded = int_str_int.decode(encoded)

I fixed it and added a test.

### CODE REVIEW

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (dahlia) [inline .cache/v/cache/lastfailed]: Could you exclude this file from commit?

File: sidertests/test_types.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 from .env import session
 from sider import types
 from sider.types import Boolean, ByteString, Date, DateTime, TZDateTime, UUID
+from sider.types import Tuple, Integer, String
```
Comment (dahlia) [inline sidertests/test_types.py]: There’s already `from sider.types import ...` line above.  I think these two lines can be merged:

```python
from sider.types import (Boolean, ByteString, Date, DateTime, Integer, String,
                         Tuple, TZDateTime, UUID)
```

File: sidertests/test_types.py [hunk 3]
```diff
@@ -112,3 +113,13 @@ def test_uuid(session):
                 b'\x11\xea.X\x97bG\xf3\xa31\xf2\xfaY\x95\xb7m', ByteString)
     with raises(ValueError):
         session.get(key('test_types_uuid'), UUID)
+
```
Comment (dahlia) [inline sidertests/test_types.py]: Although it’s trivial style, please insert one more blank line.  According [PEP 8][1]:

>  Surround top-level function and class definitions with two blank lines.

[1]: https://www.python.org/dev/peps/pep-0008/#blank-lines

File: .cache/v/cache/lastfailed (hunk 1 above)
Comment (pmuston) [inline .cache/v/cache/lastfailed]: All good suggestions.
I'll work out how to do that.
This is my first pull request.

File: sidertests/test_types.py (hunk 3 above)
Comment (pmuston) [inline sidertests/test_types.py]: Great suggestions.
I hope I have managed them successfully.
My first pull request.

**Comment by dahlia** [comment]
Thanks for your effort!

**Comment by pmuston** [comment]
Thanks for your module!!

**Pull request merged**